test/e2e/logs_basic_test.go

```diff
@@ -17,84 +17,63 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-type OutputType string
+var _ = Describe("Logs Basic", Label("logging"), Ordered, func() {
+	const (
+		mockBackendName = "log-receiver"
+		mockNs          = "log-http-output"
+		logProducerName = "log-producer-http-output" //#nosec G101 -- This is a false positive
+		pipelineName    = "http-output-pipeline"
+	)
 
-const (
-	OutputTypeHTTP   = "http"
-	OutputTypeCustom = "custom"
-)
-
-var _ = Describe("Logs Basic", Label("logging"), func() {
 	var urls = urlprovider.New()
```

Comment:
actually we do not need the urlprovider here and can only store the `telemetryExportURL` like in other tests